src/rust/catloop/mod.rs

```diff
@@ -530,6 +519,44 @@ fn pack_result(result: OperationResult, qd: QDesc, qt: u64) -> demi_qresult_t {
                 qr_value: unsafe { mem::zeroed() },
             }
         },
-        _ => panic!("Should be forwarded on to catmem"),
+        OperationResult::Push => demi_qresult_t {
```

Comment:
There are 2 Push blocks in the match, how are they differentiated?

Reply:
They don't....